internal/apitest/runner.go

```diff
@@ -25,8 +25,8 @@ import (
 var pollDefaultWait = time.Second * 180
 var pollWaitTime = time.Second * 5
 
-var unitFileName = "unit.yaml"
-var inputFileName = "input.yaml"
+var unitFileNames = []string{"unit.yaml", "unit.yml"}
+var inputFileNames = []string{"input.yaml", "input.yml"}
```

Comment:
Was wondering if this would be easier, or look better with regex? If not, can keep as is 👍

Reply:
I think this declaration would look better with regex, but then we'd have to walk the dir. Which is fine, but a little messier, imo.

Glob would be nice, too, but I don't think anything in the standard packages can do `{yml,yaml}` brace expansion.